services/QuillLMS/app/workers/google_student_classroom_worker.rb

```diff
@@ -0,0 +1,13 @@
+class GoogleStudentClassroomWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: 'critical'
+
+  def perform(student_id, context = "none")
```

Comment:
why are we passing context when it's not used?

Reply:
Updated the code without that param.